modules/basegl/src/processors/imageprocessing/imageglprocessor.cpp

```diff
@@ -47,7 +47,7 @@ ImageGLProcessor::ImageGLProcessor(std::shared_ptr<const ShaderResource> fragmen
     , dataFormat_(nullptr)
     , swizzleMask_(swizzlemasks::rgba)
     , internalInvalid_(false)
-    , shader_({{ShaderType::Fragment, fragmentShader}},
+    , shader_({{ShaderType::Vertex, utilgl::findShaderResource("img_identity.vert")}, {ShaderType::Fragment, fragmentShader}},
```

Comment:
would be nicer to just use a StringShaderResource here...

Reply:
Replaced it with a StringShaderResource.